### PR TITLE
12.0 mig purchase_blanket_order

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**

### CODE REVIEW

File: purchase_blanket_order/wizard/create_purchase_orders.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+            'view_mode': 'tree,form',
+            'res_model': 'purchase.order',
+            'view_id': False,
+            'context': {'from_purchase_order': True},
+            'type': 'ir.actions.act_window'
+        }
+
+
+class BlanketOrderWizardLine(models.TransientModel):
+    _name = 'purchase.blanket.order.wizard.line'
```
Comment (MiquelRForgeFlow) [inline purchase_blanket_order/wizard/create_purchase_orders.py:164]:
```suggestion
    _name = 'purchase.blanket.order.wizard.line'
    _description = 'Blanket Order Wizard Line'
```

**Review by DinamicheAziendali — APPROVED** [review]

**Review by andreampiovesana — APPROVED** [review]
go

**Review by marcelsavegnago — APPROVED** [review]

**Review by AdriaGForgeFlow — APPROVED** [review]
Could you take the chance to update copyrights?

Other than that code LGTM :+1:

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**